templates/mainapp/explore.html

```diff
@@ -12,7 +12,24 @@
 
 
 {% block main %}
-
+<!-----------------------------------------------------------Top Button------------------------------------------------------------------------->
+	<script>
```

Comment:
You can add this script here in [*layout.html*](https://github.com/Praful932/Kitabe/blob/91c168d1bcd5888c835fc48bf3ff3e0ac05064b7/templates/mainapp/layout.html#L357) this reduces repetition of code.